mongodbatlas/private_endpoints.go

```diff
@@ -353,3 +370,152 @@ func (s *PrivateEndpointsServiceOp) GetRegionalizedPrivateEndpointSetting(ctx co
 
 	return root, resp, err
 }
+
+// List retrieve details for all private Serverless endpoint services in one Atlas project.
+//
+// See more: https://www.mongodb.com/docs/atlas/reference/api/serverless-private-endpoints-get-all/
```

Comment:
[q] Could we stop using this doc page that will be deprecated soon and start using https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Serverless-Private-Endpoints (autogenerated with open API annotations)?

Reply:
Updated